build.gradle

```diff
@@ -17,44 +17,70 @@ plugins {
     id 'groovy'
     id 'idea'
     id 'java-gradle-plugin'
-    id "nebula.facet" version "3.4.0"
-    id 'com.gradle.plugin-publish' version '0.10.0'
+    id 'com.gradle.plugin-publish' version '0.20.0'
+}
+
+repositories {
+    mavenCentral()
 }
 
 group = 'org.gradle.crypto'
-version = '1.2.0'
+version = '1.3.0'
 
-targetCompatibility = '1.7'
-sourceCompatibility = '1.7'
+targetCompatibility = JavaVersion.VERSION_1_8
+sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
This could leverage java toolchains in order to make sure we build with the correct JDK.

Reply:
Good idea!